changelog/0.41.md

```diff
@@ -0,0 +1,44 @@
+# v0.41.0
+
+Released 2024-09-03
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+>
+> - The command usage for `ck8s` has been updated so that all commands are listed in ascending order.<br>The command `ck8s diagnostics` has a new flag to include config files in the output log file. The command usage has been changed: `ck8s diagnostics <wc|sc> [-n namespace] [--include-config]`
+> - Requires additional network policies for the pipeline. These network policies are added via the config and are generated by the network policies generator. This means that pre-defined rules can be used.
+> - Enabled matchConditions for the HNC validating webhook by default, which requires Kubernetes v1.28 or higher.
+
+## Release highlights
+
+- Added Network Policies for Tekton. If enabled, it requires the platform administrator to create pipeline specific network policies.
+- Drops support for Kubernetes v1.25, adds support for Kubernetes v1.30.<br>Nginx version upgrade to `1.25.5`.
```

Comment:
Was it meant like this?
```suggestion
- Drops support for Kubernetes v1.25, adds support for Kubernetes v1.30.
- Nginx version upgrade to `1.25.5`.
```

Reply:
Now that I have reviewed the commits I see that it the drop/support for kubernetes was because of the ingress-nginx upgrade.